Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -111,13 +111,12 @@ public async Task<Loc> GetLocationFromDevice()
         public async Task<bool> Register()
         {
             await ConfigureMobiledgeXSettings(LocationNeeded: false);
```

Comment:
LocationNeeded should be true

Reply:
Not sure why editor works...

Reply:
Fixed, Location is separated from ConfigureMobiledgeXSettings now

Reply:
Fixed